resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -746,6 +746,16 @@ sqs.send.message.deduplication_id=Message Deduplication ID:
 sqs.send.message.group_id=Message Group ID:
 sqs.send.message.send.button=Send
 sqs.send.message.success=Sent message ID: {0}
+sqs.service_name=Amazon SQS
 sqs.standard.queue.tooltip=Supports at-least-once delivery and message ordering is not preserved.
+sqs.subscribe.sns=Subscribe to SNS topic
+sqs.subscribe.sns.failed=Failed to subscribe {0} to {0}
+sqs.subscribe.sns.in_progress=Subscribing...
+sqs.subscribe.sns.select=Select SNS topic
+sqs.subscribe.sns.select.tooltip=<html><p>The SNS topics in the same AWS region.</p><p>Messages from the selected topic are sent to the queue.</p><html>
```

Comment:
I avoided phrasing it like that because the SNS topic can be from different AWS region. The reason I added the first part is to tell the user that the ResourceSelector is only listing topics in the same AWS region. If you think this is confusing, I will just remove the entire first part.

Reply:
Ah, yeah I find this confusing. More reasons we need to fixup resource selector in the future though.